### PR TITLE
Us105390 update activity to use updated organization

### PR DESCRIPTION
**Pull request by ciss1995**

### CODE REVIEW

File: components/d2l-activity-list-item/d2l-activity-list-item.js [hunk 1]
```diff
@@ -249,7 +249,7 @@ class D2lActivityListItem extends mixinBehaviors([IronResizableBehavior, D2L.Pol
 							</div>
 							<div hidden$="[[_textPlaceholder]]">
 								<h2 class="d2l-activity-list-item-title">
-									<d2l-organization-name href="[[_organizationUrl]]"></d2l-organization-name>
+									<d2l-organization-name href="[[_organizationUrl]]" token="[[token]]"></d2l-organization-name>
```
Comment (svanherk) [inline components/d2l-activity-list-item/d2l-activity-list-item.js]: I think we need to either temporarily add a `token` property, or get this element using `entity-behaviour`.  @m6jones Which would you prefer?

Comment (ciss1995) [reply components/d2l-activity-list-item/d2l-activity-list-item.js]: Yup agreed! The demo was only working when I had token as a property.

Comment (ciss1995) [reply components/d2l-activity-list-item/d2l-activity-list-item.js]: We could follow the behavior that other components are doing in that repo. quick-eval is getting it from `entity-behaviour`

Comment (svanherk) [reply components/d2l-activity-list-item/d2l-activity-list-item.js]: Yeah if you're willing to set that up here like we've been doing in the `organization` and `enrollments`, that'd be best

File: demo/d2l-activity-list-item/index.html [hunk 2]
```diff
@@ -53,17 +53,17 @@ <h3>Basic d2l-activity-list-item demo</h3>
 				<template>
 					<d2l-activity-list-item image-shimmer text-placeholder></d2l-activity-list-item>
 					<d2l-activity-list-item href="../data/base/activity.json" id="shimmer-placeholder-test" image-shimmer text-placeholder></d2l-activity-list-item>
```
Comment (ciss1995) [inline demo/d2l-activity-list-item/index.html]: Forgot to pass a token here.

File: components/d2l-activity-list-item/d2l-activity-list-item.js (hunk 1 above)
Comment (m6jones) [reply components/d2l-activity-list-item/d2l-activity-list-item.js]: So we can use the entity-behaviour for this element. I will need to update our fra when we make the change. So make sure you do a major update.

**Comment by m6jones** [comment]
Is this something we need to merge now?

**Comment by ciss1995** [comment]
@m6jones maybe? We are in the process of releasing bsi with my-courses updates. @svanherk

**Comment by svanherk** [comment]
Was another activity component added here that we need to update as well?  Or did it not use any of the organization entities?

**Pull request merged**